src/photos.cs

```diff
@@ -340,9 +340,11 @@ interface PHAssetResourceManager
 		PHAssetResourceManager DefaultManager { get; }
 
 		[Export ("requestDataForAssetResource:options:dataReceivedHandler:completionHandler:")]
+		[Async]
```

Comment:
I do not think we can Async'ify this one according to [docs](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute):

> You can use the `[Async]` attribute on methods that **return void** and whose **last** argument is a callback.